HtmlCssSample.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="HtmlCssSample.css">
+        <title>Sample Form</title>
+    </head>
+    <body>
+        <section class="FormBody">
+            <form>
+                <label for "name">Name</label>
+                
+                <input type="text" id="name" placeholder="Name" class="InputDetails">
+                
+                <label for "userName">User Name</label>
+                
+                <input type="text" id="userName" placeholder="User Name" class="InputDetails">
+                
+                <label for "passWord">Password</label>
```

Comment:
Use Pascal case for the names, Change all the places

Reply:
Resolved